CONTRIBUTING.md

```diff
@@ -42,3 +42,22 @@ git checkout -b name-of-my-branch
 git push -u origin name-of-my-branch
 ```
 10. Open a pull request
+
+
+# Running the project locally
+
+1. Cloning the repo
+2. Running the backend
+3. Running the frontend
+
+# Opening an issue
+
+1. Search through the open issues and add one if you don't find it anything related.
+2. Search through the closed/merged issues and if an issue you want to open has a merged PR, then reopen that issue with details on why you think the issue was not addressed.
+3. Add labels related to your issue
+
+# Opening a PR
```

Comment:
this should be a level 2 heading

Reply:
I've made the necessary changes plz take a look at it.